src/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>ToDo List</title>
+    <script src="https://kit.fontawesome.com/1097d30f85.js" crossorigin="anonymous"></script>
+</head>
+<body>
+  <div class="container">
+    <div class="b-bottom box">
+      <h1 class="">Today's To Do</h1>
+      <button type="button" class="refresh btn" onClick="window.location.reload();">
```

Comment:
you can create a function inside of the index.js to refresh the window of your project